### ISSUE TITLE
Error on React Native 0.42.3 or react-native-storybook-loader 1.3.0

### ISSUE THREAD

**Issue by xareelee** [elderfo/react-native-storybook-loader#24]
I run `react-native-storybook-loader` and encounter an error:

```
$ babel-node ./node_modules/.bin/rnstl 
/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:3723
  const argv = yargs(processArgs, cwd, !(function webpackMissingModule() { var e = new Error("Cannot find module \".\""); e.code = 'MODULE_NOT_FOUND';; throw e; }()))
                                                                                                                                                        ^

Error: Cannot find module "."
    at webpackMissingModule (/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:3723:84)
    at Argv (/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:3723:163)
    at Object.module.exports (/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:3718:1)
    at __webpack_require__ (/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:20:30)
    at Object.<anonymous> (/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:17048:14)
    at __webpack_require__ (/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:20:30)
    at /Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:66:18
    at Object.<anonymous> (/Users/Xaree/Development/ofo/ofoBikeIntl/node_modules/react-native-storybook-loader/dist/rnstl-cli.js:69:10)
    at Module._compile (module.js:571:32)
    at Module._extensions..js (module.js:580:10)
error Command failed with exit code 1.
```

I use `git bisect` to find that `rnstl` start to break after upgrading React Native from `0.42.0` to `0.42.3` or after upgrading react-native-storybook-loader from `1.2.0` to `1.3.0`. I have no idea why this happened.

**Comment by elderfo** [comment]
Issue with the dist version related to webpack (ref: https://github.com/yargs/yargs/issues/781). Fix is coming.

**Comment by elderfo** [comment]
Fix in: [v1.3.1](https://github.com/elderfo/react-native-storybook-loader/releases/tag/v1.3.1)
